### ISSUE TITLE
NullReferenceException in InferredIndentationDocumentOptionsProvider

### ISSUE THREAD

**Issue by miloush** [dotnet/roslyn#39709]
**Version Used**: VisualStudio.16.Preview/16.4.0-pre.3.0+29430.225

I don't have a repro (just opened a solution using jump list and VS crashed on its own), but here is a call stack:

```
Message: System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.CodeAnalysis.Editor.Implementation.Formatting.InferredIndentationDocumentOptionsProviderFactory.DocumentOptionsProvider.DocumentOptions.TryGetDocumentOption(OptionKey option, Object& value)
   at Microsoft.CodeAnalysis.Options.OptionServiceFactory.OptionService.DocumentSpecificOptionSet.GetOption(OptionKey optionKey)
   at Microsoft.CodeAnalysis.Options.OptionSet.GetOption[T](PerLanguageOption`1 option, String language)
   at Microsoft.CodeAnalysis.Editor.Implementation.Highlighting.HighlighterViewTaggerProvider.<ProduceTagsAsync>d__9.MoveNext()
```

Workarounded by starting VS first, opening the solution from MRU and quickly closing all document tabs before the crash.

Edit: I believe dumps have been collected, Report ID: 1cc02548-3359-4884-8b6a-8ff58c507053 and 9b23861d-54cf-49cc-b4a0-f8958c906574.

**Comment by jasonmalinowski** [comment]
Sorry about this one, this was fixed with #39667. We just shipped Preview 4 a few hours ago with a fix to this, so check for updates again.

**Comment by miloush** [comment]
Great, thanks!
